src/org/zaproxy/zap/extension/alert/PopupMenuItemAlert.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2016 sanchitlucknow@gmail.com
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ */
+
+package org.zaproxy.zap.extension.alert;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.Dimension;
+
+import javax.swing.JTree;
+import javax.swing.tree.DefaultMutableTreeNode;
+import javax.swing.tree.TreePath;
+
+import org.parosproxy.paros.control.Control;
+import org.parosproxy.paros.extension.ExtensionPopupMenuItem;
+import org.parosproxy.paros.core.scanner.Alert;
+import org.zaproxy.zap.extension.alert.AlertPanel;
+import org.zaproxy.zap.extension.alert.ExtensionAlert;
+
+
+public abstract class PopupMenuItemAlert extends ExtensionPopupMenuItem {
```

Comment:
- The class should override `boolean isEnableForComponent(Component)` and check that the component is the Alerts tree, to automatically show or not the menu item.
 - Also, good to allow to specify if the pop up menu item is to be used with multiple alerts or not, to have it automatically enabled or disabled (for example, see description/behaviour of `multiSelect` in `PopupMenuHttpMessageContainer(String label, boolean multiSelect)`).